ui/guest/src/classes/ICERegistry.ts

```diff
@@ -59,7 +59,13 @@ const validationChecks: { [key in ValidationKeys]: Function } = {
   allowedContentTypes() {},
   maxLength() {},
   readOnly() {},
-  required() {}
+  required() {},
+  width() {},
+  height() {},
+  minWidth() {},
+  minHeight() {},
+  maxWidth() {},
+  maxHeight() {}
```

Comment:
What were the purpose of these? How come we have so many dummy functions? @aachavarria

Reply:
That functions are used when the `drop_zone_enter` and `drop_zone_leave` events are triggered.
Right now we are only checking validation for "minCount" and "maxCount".
There are a pending ticket to add more validations: https://github.com/craftercms/craftercms/issues/4052